### PR TITLE
Update v2.93

### PR DESCRIPTION
**Pull request by LeandroStanger**
All Platforms
    Fix CVE-2018-5702 (#468)
    Fix crash on handshake if establishing DH shared secret fails (#27)
    Fix crash when switching to next tracker during announcement (#297)
    Fix potential issue during password salt extraction in OOM situation (#141)
    Workaround glib_DEFUN- and glib_REQUIRE-related configuration issue (#215)
    Fix building against OpenSSL 1.1.0+ (#24)

### CODE REVIEW

**Comment by TingPing** [comment]
bot, build

**Comment by nedrichards** [comment]
We have a patch for that CVE already that you need to remove before this will build correctly.

**Comment by wjt** [comment]
Thanks for the PR. As Nick says, we'll need to remove `0003-mitigate-dns-rebinding-attacks-against-daemon.patch`, and also add the new release to the appdata.

File: com.transmissionbt.Transmission.appdata.xml [hunk 1]
```diff
@@ -157,7 +157,7 @@
         <p>Fix CVE-2018-5702 security vulnerability (see https://bugs.chromium.org/p/project-zero/issues/detail?id=1447 and https://github.com/transmission/transmission/pull/468 for more details).</p>
       </description>
     </release>
-    <release date="2017-12-13" version="2.92-3">
+    <release date="2018-01-24" version="2.93">
```
Comment (wjt) [inline com.transmissionbt.Transmission.appdata.xml]: Please add a new entry with the new news.

File: 0003-mitigate-dns-rebinding-attacks-against-daemon.patch [hunk 2]
```diff
@@ -1,286 +0,0 @@
-From 440f03e9ad5ed3efc9e7d943befe33e5cbb65a20 Mon Sep 17 00:00:00 2001
```
Comment (wjt) [inline 0003-mitigate-dns-rebinding-attacks-against-daemon.patch]: You'll also need to remove the reference to this file from the JSON. Have you tried building this branch?

File: com.transmissionbt.Transmission.appdata.xml (hunk 1 above)
Comment (wjt) [reply com.transmissionbt.Transmission.appdata.xml]: I meant to add a new entry instead of changing the date & version on the old entry.

**Comment by wjt** [comment]
Thanks for the PR! In the end I reverted your two patches modifying appdata and applied my own version that includes the release notes: dd076b0941e46b1943d8f02d6f4027fa030f6bb3

**Pull request merged**